website/docs/r/cis_logpush_jobs.html.markdown

```diff
@@ -0,0 +1,41 @@
+---
```

Comment:
This document is for resource but it show the same content of datasource

Reply:
changed